test/matrices.jl

```diff
@@ -0,0 +1,10 @@
+@testset "Matrices" begin
+  # tests with Aitchson's definitions
+  d = 10
+  D = d+1
+  @test JColumn(d) == ones(d)
+  @test JMatrix(d) == ones(d, d)
```

Comment:
I would rewrite these in terms of the shortcuts `J` and `j` after exporting these names.